frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -0,0 +1,182 @@
+import { FluentVariable } from "@fluent/bundle";
+import { Localized, useLocalization } from "@fluent/react";
+import {
+  getBundlePrice,
+  getBundleSubscribeLink,
+  isBundleAvailableInCountry,
+} from "../../functions/getPlan";
+import { isFlagActive } from "../../functions/waffle";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+import { MozillaVpnWordmark } from "../Icons";
+import styles from "./BundleBanner.module.scss";
+import { LinkButton } from "../Button";
+import womanInBanner400w from "./images/bundle-banner-woman-400w.png";
+import womanInBanner768w from "./images/bundle-banner-woman-768w.png";
+import womanInBanner1280w from "./images/bundle-banner-woman-1280w.png";
+import bundleFloatOne from "./images/bundle-float-1.svg";
+import bundleFloatTwo from "./images/bundle-float-2.svg";
+import bundleFloatThree from "./images/bundle-float-3.svg";
+import bundleLogo from "./images/vpn-and-relay-logo.svg";
+import { trackPurchaseStart } from "../../functions/trackPurchase";
+
+export type Props = {
+  runtimeData: RuntimeData;
+};
+
+type FloatingFeaturesProps = {
+  icon: string;
+  text: string;
+  position: string;
+  vars?: Record<string, FluentVariable>;
+};
+
+const FloatingFeatures = (props: FloatingFeaturesProps) => {
+  const { l10n } = useLocalization();
+
+  const text = props.vars ? (
+    <Localized id={props.text} vars={props.vars}>
+      <span className={styles["float-features-text"]} />
+    </Localized>
+  ) : (
+    <span className={styles["float-features-text"]}>
+      {l10n.getString(props.text)}
```

Comment:
It doesn't really matter, but good to know: you can also pass vars to `getString`:

```js
      {l10n.getString(props.text, props.vars)}
```

Reply:
Good to know! I was trying to find examples.